Makefile

```diff
@@ -38,7 +38,7 @@ all-apps: $(APPS)
 bundle-%: clone-%
 	$(GOVUK_DOCKER) build $*-lite
 	$(GOVUK_DOCKER) run $*-lite rbenv install -s || ($(GOVUK_DOCKER) build --no-cache $*-lite; $(GOVUK_DOCKER) run $*-lite rbenv install -s)
-	if [ -f Gemfile.lock ]; then $(GOVUK_DOCKER) run $*-lite sh -c 'gem install --conservative --no-document bundler -v $$(grep -A1 "BUNDLED WITH" Gemfile.lock | tail -1)'; fi
+	if [ -f "${GOVUK_ROOT_DIR}/$*/Gemfile.lock" ]; then $(GOVUK_DOCKER) run $*-lite sh -c 'gem install --conservative --no-document bundler -v $$(grep -A1 "BUNDLED WITH" Gemfile.lock | tail -1)'; fi
```

Comment:
humm interesting, so that would do it without using the govuk-docker run command, right?

Is there any advantage to that other than just a shorter line?
I'm guessing if they've got thsi far we can guarentee they have govuk-docker installed, whereas we don't know much about if they have ruby perhaps?

Reply:
It's only a minor suggestion. There are two (minor) reasons for doing the check within the container:

- That's where we're looking for the `Gemfile.lock` (as part of the `grep`).
- We don't need to duplicate [knowledge](https://github.com/alphagov/govuk-docker/blob/master/projects/collections-publisher/docker-compose.yml#L11) of where `Gemfile.lock` is on the host system.